docs/rules/array-callback-return.md

```diff
@@ -72,6 +72,21 @@ var foo = Array.from(nodes, function(node) {
 var bar = foo.map(node => node.getAttribute("id"));
 ```
 
+## Options
+
+This rule has an object option:
+
+* `"allowImplicit": false` (default) disallows implicitly returning undefined with a return; statement.
```

Comment:
Nitpick: Can `undefined` and `return;` be wrapped in backticks so it's clear that they're code snippets? Also, `return` probably doesn't need the semicolon.